scripts/utils/cli-step.ts

```diff
@@ -23,6 +23,7 @@ export const steps = {
       output: { type: 'string' },
       // TODO allow default values for strings
       branch: { type: 'string', values: ['next'] },
+      init: { type: 'boolean', inverse: true },
```

Comment:
maybe this should be called `useLocalCli` ?

Reply:
I think this is an existing option @yannbf